shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -643,6 +643,9 @@ fun geoPropertyToWKT(jsonFragment: Map<String, Any>): Map<String, Any> {
             val geoAttribute = jsonFragment[geoProperty] as MutableMap<String, Any>
             val geoJsonAsString = geoAttribute[JSONLD_VALUE]
             val wktGeom = geoJsonToWkt(geoJsonAsString!! as Map<String, Any>)
+                .fold({
+                    throw BadRequestDataException(it.message)
```

Comment:
it's not possible to do without the throw?

Reply:
`JsonLdUtils` is unfortunately still mainly "exception-based". More specifically about this function, it is called from `doJsonLdExpansion` which already throws exception. Did not want to mix both concepts (nor to add more changes to this already quite big PR).